bigquery/main.go

```diff
@@ -283,18 +283,27 @@ func (bq *actualBQ) EnsureDataset(ctx context.Context, datasetName string) error
 func (bq *actualBQ) EnsureTable(ctx context.Context, tableName string) error {
 	// Check for existence of table, create if false
 	bq.table = bq.dataset.Table(tableName)
-	_, err := bq.dataset.Table(tableName).Metadata(ctx)
+	schema, err := bigquery.InferSchema(bqRow{})
```

Comment:
Can you try to test this ensuring functionality?

Reply:
tested!